api/api/accounts/tests.py

```diff
@@ -0,0 +1,170 @@
+from collections import OrderedDict
+
+import pytest
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.test import APIClient
+
+User = get_user_model()
+user_creation_endpoint = '/users/'
+primary_test_user_mail = 'test@user.com'
+
+
+@pytest.mark.django_db
+def testCreateUserView_testPermission_nonAdminRequest_throwsForbidden():
+    # given
+    user = User.objects.create(
+        email=primary_test_user_mail, first_name='Test', last_name='User')
+    client = APIClient()
```

Comment:
nit: we can use `pytest.fixture` to create repetitive objects once.

```python
@pytest.mark.django_db
@pytest.fixture(scope='function')
def admin_user(django_user_model):
    return django_user_model.objects.create(email=primary_test_user_mail, first_name='Test', last_name='User', is_staff=True)

@pytest.fixture(scope='function')
def client():
    return APIClient()
```

And then use it like:
```python
@pytest.mark.django_db
def testCreateUserView_testPermission_nonAdminRequest_throwsForbidden(admin_user, client):
    ...
```

`pytest_django` also allows you to use `django_user_model` fixture to get `User` model: [Reference](https://pytest-django.readthedocs.io/en/latest/helpers.html#django-user-model)